internal/search/query/validate.go

```diff
@@ -151,6 +151,13 @@ func validateField(field, value string, negated bool, seen map[string]struct{})
 		return nil
 	}
 
+	isNumber := func() error {
+		if _, err := strconv.Atoi(value); err != nil {
+			return fmt.Errorf("field %s has value %s, %s is not a number", field, value, value)
```

Comment:
```suggestion
			return fmt.Errorf("field %s has value %[2]s, %[2]s is not a number", field, value)
```

https://play.golang.org/p/ldLai01i41W

Reply:
![](https://media1.giphy.com/media/tCV2LrPPYfqCY/giphy.gif?cid=5a38a5a27ba4daae4568847c8c05f0ada08a14b4c28af0c9&rid=giphy.gif)